.github/workflows/odh_notebook_controller_integration_test.yaml

```
@@ -82,12 +82,10 @@ jobs:
         run: |
           set -x
 
-          cd components/odh-notebook-controller/config/default
+          cd components/odh-notebook-controller/config/base
           kubectl create ns opendatahub
 
-          export CURRENT_NOTEBOOK_IMG=quay.io/opendatahub/odh-notebook-controller:latest
-          export PR_NOTEBOOK_IMG=localhost/${{env.IMG}}:${{env.TAG}}
-          kustomize edit set image ${CURRENT_NOTEBOOK_IMG}=${PR_NOTEBOOK_IMG}
+          echo "odh-kf-notebook-controller-image=localhost/${{env.IMG}}:${{env.TAG}}" > params.env
```

```suggestion
          echo "odh-notebook-controller-image=localhost/${{env.IMG}}:${{env.TAG}}" > params.env
```

-kf- is the other controller 😮‍💨